### PR TITLE
.htaccess file

### PR DESCRIPTION
**Pull request by GiridharUrkude**
created .htaccess file

### CODE REVIEW

File: agrion/.htaccess [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+RewriteCond %{HTTP_ACCEPT} text/\* [OR]
+RewriteCond %{HTTP_ACCEPT} \*/\* [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+#RewriteRule ^asteroids$ http://dgarijo.github.io/VideoGameOntology/GameExtensions/AsteroidsOntology/asteroidsDoc/  [R=303,NE,L]
+#RewriteCond %{HTTP_ACCEPT} ^.*application/rdf\+xml.* 
+#RewriteRule ^asteroids$ http://dgarijo.github.io/VideoGameOntology/GameExtensions/AsteroidsOntology/asteroids.owl [R=303,NE,L]
+#RewriteCond %{HTTP_ACCEPT} ^.*text/turtle.* 
+#RewriteRule ^asteroids$ http://dgarijo.github.io/VideoGameOntology/GameExtensions/AsteroidsOntology/asteroids.ttl [R=303,NE,L]
+#default response: owl
+RewriteRule ^asteroids$ http://linkedsensor.in/ontology/agrion.owl [R=303,NE,L]
```
Comment (davidlehn) [inline agrion/.htaccess:25]: Looks like a copy & paste issue here.

File: agrion/readme.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Agriculture Ontology
```
Comment (davidlehn) [inline agrion/readme.md:1]: Please add maintainer details (github username and/or email).

File: linkedsensor/ontology/.htaccess [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+
+#RewriteRule ^asteroids$ http://dgarijo.github.io/VideoGameOntology/GameExtensions/AsteroidsOntology/asteroidsDoc/  [R=303,NE,L]
+#RewriteCond %{HTTP_ACCEPT} ^.*application/rdf\+xml.* 
+#RewriteRule ^asteroids$ http://dgarijo.github.io/VideoGameOntology/GameExtensions/AsteroidsOntology/asteroids.owl [R=303,NE,L]
+#RewriteCond %{HTTP_ACCEPT} ^.*text/turtle.* 
+#RewriteRule ^asteroids$ http://dgarijo.github.io/VideoGameOntology/GameExtensions/AsteroidsOntology/asteroids.ttl [R=303,NE,L]
+
+#default response: owl
+RewriteRule ^asteroids$ http://linkedsensor.in/ontology/agrion.owl [R=303,NE,L]
```
Comment (davidlehn) [inline linkedsensor/ontology/.htaccess:27]: Similar copy & paste issue.

File: linkedsensor/readme.md [hunk 4]
```diff
@@ -0,0 +1 @@
+
```
Comment (davidlehn) [inline linkedsensor/readme.md:1]: Please add a quick note and maintainer info here.

File: linkedsensor/ontology/readme.md [hunk 5]
```diff
@@ -0,0 +1 @@
+
```
Comment (davidlehn) [inline linkedsensor/ontology/readme.md:1]: Is this file needed?

**Comment by davidlehn** [comment]
Would be nice if you could squash all these commits.  There is some unneeded commit churn here.

File: linkedsensor/ontology/readme.md (hunk 5 above)
Comment (GiridharUrkude) [reply linkedsensor/ontology/readme.md]: the file is deleted..

File: linkedsensor/ontology/.htaccess (hunk 3 above)
Comment (davidlehn) [reply linkedsensor/ontology/.htaccess:20]: I'm guessing "asteroids" came from "games/.htaccess"
I don't know what mapping you want here.  Those source patterns are relative to the current directory, so in this case:
https://w3id.org/linkedsensor/ontology/asteroids => http://linkedsensor.in/ontology/agrion.owl
Maybe you want
```
RewriteRule ^$ http://linkedsensor.in/ontology/agrion.owl [R=303,NE,L]
```
or
```
RewriteRule ^agrion$ http://linkedsensor.in/ontology/agrion.owl [R=303,NE,L]
```
Did you want to use https on your site?

Comment (GiridharUrkude) [reply linkedsensor/ontology/.htaccess:20]: ok, I got it. thank you for your help.

File: agrion/.htaccess (hunk 1 above)
Comment (GiridharUrkude) [reply agrion/.htaccess]: This file is deleted. It was created for the experimental purpose.

**Comment by GiridharUrkude** [comment]
Dear Admin please accept the pull request.

**Comment by GiridharUrkude** [comment]
Dear Admin, please accept the pull request...

**Comment by GiridharUrkude** [comment]
thank you very much.
